TESTS.md

```diff
@@ -1738,8 +2047,6 @@ schroffl Hindent produces invalid Syntax from FFI exports #479
 -- https://github.com/commercialhaskell/hindent/issues/479
 foreign export ccall "test" test :: IO ()
 
-foreign import ccall "test" test :: IO ()
-
```

Comment:
`ghc-lib-parser` doesn't have the information of whether the `safe` keyword is omitted or not. Thus, we can't test this line.